README.md

```diff
@@ -24,24 +24,12 @@ with Configurator() as config:
     config.include("pyramid_sanity")
 ```
 
-Settings
---------
-
 By default all fixes are enabled. You can disable them individually with settings:
 
 ```python
 config.add_settings({
     # Don't check for badly declared forms.
-    "pyramid_sanity.check_form": False,
-
-    # Don't check for badly encoded query params.
-    "pyramid_sanity.check_params": False,
-
-    # Don't check for badly encoded URL paths.
-    "pyramid_sanity.check_path": False,
-
-    # Don't safely encode redirect locations.
-    "pyramid_sanity.ascii_safe_redirects": False
+    "pyramid_sanity.check_form": False
```

Comment:
I shifted this stuff about for a few reasons:

 * This section is about disabling things, it didn't make sense to list the `pyramid_sanity.check_form.assume_form_on_blank` here
* This is more usage than documentation of the settings. It was trying to do double duty, but I've split them up